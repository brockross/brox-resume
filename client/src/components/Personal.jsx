import React from 'react';

const Personal = (props) => {
  return (
    <div>
      <h3>Personal</h3>
      <p>dad | musician | writer | thinker | human-lover</p>
    </div>
  )
}

export default Personal;