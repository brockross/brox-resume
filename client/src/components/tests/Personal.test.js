import React from 'react';
import { shallow } from 'enzyme';

import Personal from  '../Personal.jsx';

describe('Personal', () => {
  it('should render correctly', () => {
    const component = shallow(<Personal/>);

    expect(component).toMatchSnapshot();
  });
});