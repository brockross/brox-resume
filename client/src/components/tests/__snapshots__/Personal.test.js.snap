// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Personal should render correctly 1`] = `
<div>
  <h3>
    Personal
  </h3>
  <p>
    dad | musician | writer | thinker | human-lover
  </p>
</div>
`;
